docs/readme.md

```diff
@@ -82,15 +82,38 @@ log.setLevel(flags.logLevel);
 launchChromeAndRunLighthouse('https://example.com', flags).then(...);
 ```
 
+## Configuration
+In order to configure Lighthouse programmatically, you need to pass the config object as the 3rd argument.
```

Comment:
> you need to pass the config object as the 3rd argument

The config parameter isn't required (if not provided, it defaults to the default config), so maybe this needs more emphasis on it being an option if needing a more custom LH run?

Reply:
I've clarified the fact if omitted, a default config will be used.